cfgov/unprocessed/js/organisms/FilterableListControls.js

```diff
@@ -72,7 +77,52 @@ function FilterableListControls( element ) {
    * Initialize FilterableListControls events.
    */
   function _initEvents() {
-    _form.addEventListener( 'submit', _formSubmitted );
+    var labelDom = _dom.querySelector( '.o-expandable_label' );
+    var label;
+
+    if ( labelDom ) {
+      label = labelDom.textContent.trim();
+    }
+
+    _expandable.addEventListener( 'expandBegin',
+    function sendEvent( ) {
+      Analytics.sendEvent( 'Filter:open', label );
+    } );
+
+    _expandable.addEventListener( 'collapseBegin',
+    function sendEvent( ) {
+      Analytics.sendEvent( 'Filter:close', label );
+    } );
+
+    _form.addEventListener( 'change', function sendEvent( event ) {
```

Comment:
Or I guess it would be `_formChangeHandler` or something. I'd err on moving the event handler functions outside of the `addEventListener` call so that the listeners can be stacked together and function can be jsdoc annotated and shared, if needed.